inference-engine/src/gna_plugin/gna_device.cpp

```diff
@@ -476,6 +489,16 @@ void GNADeviceHelper::dumpXnnForDeviceVersion(
     outStream.write("Gna2ModelSueCreekHeader", 24);
     outStream.write(reinterpret_cast<const char*>(&sueHeader), sizeof(sueHeader));
 }
+
+void GNADeviceHelper::createVirtualDevice(Gna2DeviceVersion devVersion, std::string purpose) {
+    const auto status = Gna2DeviceCreateForExport(devVersion, &nGnaDeviceIndex);
+    GNADeviceHelper::checkGna2Status(status, "Gna2DeviceCreateForExport(" + std::to_string(devVersion) + ")" + purpose);
+}
+
+void GNADeviceHelper::detectGnaDeviceVersion() {
```

Comment:
Maybe `updateGnaDeviceVersion` is the more clear name for it

Reply:
OK